fish-finder_2019-08-05.sql

```diff
@@ -0,0 +1,56 @@
+# ************************************************************
+# Sequel Pro SQL dump
+# Version 4541
+#
+# http://www.sequelpro.com/
+# https://github.com/sequelpro/sequelpro
+#
+# Host: 192.168.20.20 (MySQL 5.6.44)
+# Database: fish-finder
+# Generation Time: 2019-08-05 10:16:32 +0000
+# ************************************************************
+
+
+/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
+/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
+/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
+/*!40101 SET NAMES utf8 */;
+/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
+/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
+/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;
+
+
+# Dump of table fish
+# ------------------------------------------------------------
+
+DROP TABLE IF EXISTS `fish`;
+
+CREATE TABLE `fish` (
+  `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
+  `name` varchar(255) DEFAULT '',
+  `species` varchar(255) DEFAULT '',
+  `img-filepath` varchar(255) DEFAULT '',
+  `length` int(11) unsigned NOT NULL,
+  `agression` int(11) unsigned NOT NULL,
```

Comment:
you mentioned max values, can these be implemented in mySQL?

Reply:
Sort of! Resulved